### DISCUSSION TITLE
I work on open source projects, but do not have free access to GitHub Copilot

### DISCUSSION THREAD

**Discussion by NiuXiangQian** [community/community#65402] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I maintain a small open source project with 160+stars, but still don't qualify for free. Why?

[chatgpt-stream](https://github.com/NiuXiangQian/chatgpt-stream)

**Comment by lab596** [comment, marked as answer]
This is what it says on their website: 

"People who maintain popular open source projects receive a credit to have 12 months of GitHub Copilot access for free. A maintainer of a popular open source project is defined as someone who has write or admin access to one or more of the most popular open source projects on GitHub. Simply visit the GitHub Copilot subscription page to see if you are one of the open-source maintainers that meet our criteria for a complimentary subscription. If you do, you should see that you can add GitHub Copilot for no charge. If you see a charge on the purchase page then this means that you do not qualify at this time. Once awarded, if you are still a maintainer of a popular open source project when your initial 12 months subscription expires then you will be able to renew your subscription for free."

As you can see their description is extremely vague and doesn't provide any specific boxes that need to be checked. However, when digging online it looks like the general consensus is a project that has around 5k-10k stars.  Sorry, I cant find a more specific answer. Let me know if this helps!

**Reply by NiuXiangQian** [reply]
I had another open source project with 20 stars when Copilot was first released, and Copilot allowed me to use it for free for about three months. It's a pity that the requirements are raised so high now.
